Volume/Views/BrowserView.swift

```diff
@@ -111,8 +111,9 @@ struct BrowserView: View {
 
     func displayShareScreen() {
         if let articleUrl = article.articleUrl {
-            // TODO: Allow users to share "DOWNLOAD VOLUME LINK" when we push to AppStore
-            let shareVC = UIActivityViewController(activityItems: [articleUrl], applicationActivities: nil)
+            let downloadURL = URL(string: Secrets.downloadLink)
+            let shareItems: [Any] = [articleUrl, downloadURL ?? ""]
```

Comment:
Can there be consistency between `articleUrl` and `downloadURL` in terms of capitalization

Reply:
And just everywhere that `url` appears in the codebase.

Reply:
This actually won't matter once I get the deep linking PR out which should be within a few hours lol

Reply:
But yeah I'll make it more consistent